docs/04-duplicating-translated-documents.md

```diff
@@ -0,0 +1,29 @@
+# Duplicating translated documents
+
+This plugin includes a custom document action that will duplicate a localized document, its translations, and the metadata document that relates them.
+
+![Duplicate document action](./img/duplicate-document-action.png)
+
+Import the document action and configure which document types will use it:
+
+```ts
+import {
+  documentInternationalization,
+  DuplicateWithTranslationsAction,
+} from '@sanity/document-internationalization'
+
+export default defineConfig({
+  // ...all other config
+  document: {
+    actions(prev, { schemaType }) {
+      return schemaTypes.includes(schemaType)
```

Comment:
Maybe add a comment here that `schemaTypes` would be the same array that you add to the `documentInternationalization` plugin config. When you copy-paste this snippet it's likely an undefined variable.